presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlJdbcConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mysql;
+
+import com.mysql.jdbc.Driver;
+import io.prestosql.plugin.jdbc.BaseJdbcConfig;
+
+import javax.validation.constraints.AssertTrue;
+
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class MySqlJdbcConfig
+        extends BaseJdbcConfig
```

Comment:
i feel it's better to avoid extending Config classes. can we avoid this?

Reply:
I'm not sure how the config system would handle multiple bound config classes that have conflicting annotations like `ConfigDescription`. Also, we want this to be identical to the parent, but with additional validations, so this seems the best way.

Ideally, there would be some way to bind additional bean validators.